recipes/mosaic/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "mosaic" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/E3SM-Project/mosaic/archive/{{ version }}.tar.gz
+  sha256: 6393c3bc58257c1214903e30f19cdf7688ba6262215088655e77d9c1e852d57e
+
+build:
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  noarch: python
+  number: 0
+
+requirements:
+  host:
+    - python >=3.9
+    - pip
+  run:
+    - python >=3.9
+    - cartopy
+    - cmocean
+    - h5netcdf
+    - netcdf4
+    - matplotlib-base
+    - numpy
+    - pyproj
+    - scipy
+    - xarray
+
+test:
+  imports:
+    - mosaic
+  requires:
+    - pip
+  commands:
+    - pip check
```

Comment:
@andrewdnolan, ideally, we will add more rigorous testing in the future but this should be adequate for now.

Reply:
I'm working on [some documentation](https://github.com/andrewdnolan/mosaic/tree/docs) now, but tests will be the nest task to tackle!